src/main/java/Duke.java

```diff
@@ -1,10 +1,30 @@
+import java.util.Scanner;
+
 public class Duke {
     public static void main(String[] args) {
-        String logo = " ____        _        \n"
-                + "|  _ \\ _   _| | _____ \n"
-                + "| | | | | | | |/ / _ \\\n"
-                + "| |_| | |_| |   <  __/\n"
-                + "|____/ \\__,_|_|\\_\\___|\n";
-        System.out.println("Hello from\n" + logo);
+        // String logo = " ____        _        \n"
+        //         + "|  _ \\ _   _| | _____ \n"
+        //         + "| | | | | | | |/ / _ \\\n"
+        //         + "| |_| | |_| |   <  __/\n"
+        //         + "|____/ \\__,_|_|\\_\\___|\n";
+        // System.out.println("Hello from\n" + logo);
+        // String indent = "    ";
+        // String newLine = indent + "____________________________________________________________ \n";
+        // String greeting = indent + "Good day good sir! I am Chatimous Maximous, here to help you with your every need!\n";
+        // System.out.println(newLine + greeting);
+
+        ChatBot Chatty = new ChatBot();
+        Scanner reader = new Scanner(System.in);
+        String input = reader.nextLine();
+        String message;
```

Comment:
Hi, small issue but the coding standard, under the 'Variables' section states that variables should be initialized where they are declared 😢 Personally, I also make the same mistake but just take note in the future 👍